ws-nextjs-app/pages/[service]/send/[id]/FormField/InvalidMessageBox.tsx

```diff
@@ -46,37 +46,53 @@ const formatValidationMessage = (
   return message;
 };
 
-export default ({
-  id,
-  messageCode,
-  hasArrowStyle = true,
-  suffix,
-  validation,
-}: InvalidMessageBoxProps) => {
-  const {
-    translations: { ugc = fallbackTranslations },
-  } = useContext(ServiceContext);
+const InvalidMessageBox = forwardRef(
+  (
+    {
+      id,
+      messageCode,
+      hasArrowStyle = true,
+      isErrorSummary = false,
+      suffix,
+      validation,
+    }: InvalidMessageBoxProps,
+    ref: ForwardedRef<HTMLElement>,
+  ) => {
+    const {
+      translations: { ugc = fallbackTranslations },
+    } = useContext(ServiceContext);
 
-  const message = formatValidationMessage(
-    ugc[messageCode ?? InvalidMessageCodes.FieldRequired] ?? '',
-    validation,
-  );
+    const message = formatValidationMessage(
+      ugc[messageCode ?? InvalidMessageCodes.FieldRequired] ?? '',
+      validation,
+    );
 
-  return (
-    <>
-      {hasArrowStyle && <div css={styles.errorArrow} />}
-      <div css={styles.errorMessageBox(hasArrowStyle)}>
-        <ErrorSymbol />
-        <Paragraph
-          id={id}
-          css={styles.errorText}
-          fontVariant="sansBold"
-          size="minion"
+    return (
+      <>
+        {hasArrowStyle && <div css={styles.errorArrow} />}
+        <div
+          css={[styles.errorMessageBox, !hasArrowStyle && styles.hasArrowStyle]}
         >
-          {message}
-          <VisuallyHiddenText>{`, ${suffix}`}</VisuallyHiddenText>
-        </Paragraph>
-      </div>
-    </>
-  );
-};
+          <ErrorSymbol />
+          <Text
+            id={id}
+            css={[
+              styles.errorText,
+              isErrorSummary && styles.focusIndicatorErrorSummary,
+            ]}
+            fontVariant="sansBold"
+            size="minion"
+            as="strong"
+            {...(isErrorSummary && { tabIndex: -1 })}
+            {...(ref && { ref })}
+          >
+            {message}
+            <VisuallyHiddenText>{`, ${suffix}`}</VisuallyHiddenText>
```

Comment:
What's the visually hidden text for?

Reply:
I also see we have this for other error messages, should it be there? Take a look without CSS...

Reply:
I believe this is if the user is navigating through the form by arrows (not form controls), so it is then clear which field the error message corresponds to. I can check with UX?

Reply:
Is it on the screen reader UX?

Reply:
It was implemented in this PR: https://github.com/bbc/simorgh/pull/11726.

Reply:
I don't think it should be there @shayneahchoon @emilysaffron we have the aria describedby for the association

Reply:
Hello, @greenc05 we tried adding in the aria-described-by but it it didn't work as we would like. Have added a fix to address the issue. So now the visually hidden text will only appear on generic messages for inputs and NOT on the summary box at all.